.github/styles/alex/Condescending.yml

```diff
@@ -0,0 +1,16 @@
+---
+extends: existence
+message: Using '%s' may come across as condescending.
+link: https://css-tricks.com/words-avoid-educational-writing/
+level: error
```

Comment:
should be `warning` not error. This entire file in laughable. The vast majority of technical content use these terms in a non-condescending way, e.g. "It is not obvious how to easily create a new ML project from scratch. We attempt to make this more simple here" throws 3 errors really?